QUERIES.md

```diff
@@ -1,4 +1,4 @@
-The top level name is called a measurement. These names can contain any characters. Then there are field names, field values, tag keys and tag values, which can also contain any characters. However, if the measurement, field, or tag contains any character other than [A-Z,a-z,0-9,_], or if it starts with a digit, it must be double-quoted. Therefore anywhere a measurement name, field name, field value, tag name, or tag value appears it should be wrapped in double quotes.
+The top level name is called a measurement. These names can contain any characters. Then there are field names, field values, tag keys and tag values, which can also contain any characters. However, if the measurement, field, or tag contains any character other than [A-Z,a-z,0-9,_], or if it starts with a digit, it must be double-quoted. Therefore anywhere a measurement name, field key, field value, tag key, or tag value appears it should be wrapped in double quotes.
```

Comment:
> Therefore anywhere a measurement name, field key, field value, tag key, or tag value appears it should be wrapped in double quotes.

This doesn't sound right. Tag values, field values, and passwords are interpreted as string literals and should be single quoted. Measurement names, tag keys, field keys, and any other names are identifiers. Identifiers are double quoted unless they fit certain conditions.